app/ui.js

```diff
@@ -1564,15 +1548,14 @@ const UI = {
         }
 
         const blist = [0, 1,2,4];
-        for (let b = 0; b < blist.length; b++) {
-            const button = document.getElementById('noVNC_mouse_button' +
-                                                 blist[b]);
-            if (blist[b] === num && !view_only) {
+        blist.forEach((b) => {
+            const button = document.getElementById('noVNC_mouse_button' + b);
```

Comment:
template literal

Reply:
Yup, there will be a future commit converting all string concatenations to string literals.
Just trying to go little by little :)